targettests/execution/tensor.cpp

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// REQUIRES: c++20
+
+// clang-format off
+// RUN: clang++ -std=c++20 \
```

Comment:
I don't think we need to test a `clang++` command line. Tests should run against an installation. Do all these include paths (which look valid for in-tree) exist in an installation?